docs/docs/sample-scripts/spo/remove-custom-themes.md

```diff
@@ -1,3 +1,9 @@
+---
+tags:
+  - Provisioning
```

Comment:
I don't think `provisioning` applies here as this script is more about cleaning up unneeded resources